app/views/layouts/_base.html.erb

```diff
@@ -26,6 +26,29 @@
   <meta http-equiv="cleartype" content="on"><![endif]-->
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta name="HandheldFriendly" content="True">
+
+  <meta name="application-name" content="Money Advice Service">
+  <meta name="msapplication-starturl" content="https://www.moneyadviceservice.org.uk">
+  <meta name="msapplication-square70x70logo" content="/favicon_ms_70x70.png">
+  <meta name="msapplication-square150x150logo" content="/favicon_ms_150x150.png">
+  <meta name="msapplication-wide310x150logo" content="/favicon_ms_310x150.png">
+  <meta name="msapplication-square310x310logo" content="/favicon_ms_310x310.png">
+  <%= tag 'link', rel: 'icon', type: 'image/png', sizes: '32x32', href: '/favicon_32x32.png' %>
+  <%= tag 'link', rel: 'icon', type: 'image/png', sizes: '48x48', href: '/favicon_48x48.png' %>
+  <%= tag 'link', rel: 'icon', type: 'image/png', sizes: '72x72', href: '/favicon_72x72.png' %>
+  <%= tag 'link', rel: 'icon', type: 'image/png', sizes: '96x96', href: '/favicon_96x96.png' %>
+  <%= tag 'link', rel: 'icon', type: 'image/png', sizes: '128x128', href: '/favicon_128x128.png' %>
+  <%= tag 'link', rel: 'icon', type: 'image/png', sizes: '144x144', href: '/favicon_144x144.png' %>
```

Comment:
This line can be deleted (144px icon unique to Android, referenced via manifest)